metricbeat/module/elasticsearch/pending_tasks_summary/_meta/fields.yml

```diff
@@ -0,0 +1,21 @@
+- name: cluster.pending_tasks_summary
```

Comment:
Could we split this second metricset in a separate PR?

Reply:
Hi, I split this PR into two PRs :

- https://github.com/elastic/beats/pull/7144
- https://github.com/elastic/beats/pull/7145